godtools/App/Features/ToolDetails/Presentation/ToolDetails/Views/VersionsCard/ToolDetailsVersionsCardView.swift

```diff
@@ -10,63 +10,113 @@ import SwiftUI
 
 struct ToolDetailsVersionsCardView: View {
     
+    private let bannerHeight: CGFloat = 87
+    
+    @ObservedObject var viewModel: ToolDetailsVersionsCardViewModel
+    
     @State var isSelected: Bool = false
     
     var body: some View {
         
-        VStack {
+        ZStack {
             
-            Image("tutorial_tool_non_english")
+            Color.white
```

Comment:
The white card background looks pretty similar to `RoundedCardBackgroundView`, could use that here to add the background/shadow instead.

Reply:
Only problem I noticed when using a RoundedRectangle() is the top of the card doesn't get rounded because of the image.

![top-not-rounded-clipped](https://user-images.githubusercontent.com/59846460/176469010-5a4cc59d-e154-48dd-9056-4e84771beae2.png)

Reply:
I think it's because of the image anyway.